Add service manual links to README (#504)
alphagov/accessible-autocomplete: README.md

- **Accessibility**: Following WAI-ARIA best practices and testing with assistive technologies.
- **User experience**: Supporting a wide variety of user needs.

> - **Compatibility**: Working with as many browsers, devices, and assistive technologies as possible. However, we only support versions of Internet Explorer (IE) from IE11 on.

Author: ```suggestion
- **Compatibility**: Working with [recommended browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in) and [assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#which-assistive-technologies-to-test-with).
```

- **Accessibility**: Following WAI-ARIA best practices and testing with assistive technologies.
- **User experience**: Supporting a wide variety of user needs.

> - **Compatibility**: Working with as many browsers, devices, and assistive technologies as possible. However, we're unable to support versions of Internet Explorer (IE) before IE11.

Author: Would this wording be less negative (because it says what we're able to do, instead of what we're unable to do)?

```suggestion
- **Compatibility**: Working with as many browsers, devices, and assistive technologies as possible. However, we only support versions of Internet Explorer (IE) from IE11 on.
```
